docs/documentation/case.md

```diff
@@ -372,6 +358,16 @@ Details of implementation of viscosity in MFC can be found in [Coralic (2015)](r
 | `wave_speeds`          | Integer | Wave-speed estimation: [1] Direct (Batten et al. 1997); [2] Pressure-velocity* (Toro 1999)	 |
 | `weno_Re_flux`         | Logical | Compute velocity gradient using scaler divergence theorem	 |
 | `weno_avg`          	 | Logical | Arithmetic mean of left and right, WENO-reconstructed, cell-boundary values |
+| `dt`                   | Real    | Time step size |
+| `t_step_start`         | Integer | Simulation starting time step |
+| `t_step_stop`          | Integer | Simulation stopping time step |
+| `t_step_save`          | Integer | Frequency to output data |
+| `t_step_print`         | Integer | Frequency to print the current step number to standard output (default 1) |
+| `cfl_dt`               | Logical | CFL based adaptive time-stepping |
+| `cfl`                  | Real    | Specified CFL value |
+| `n_start`              | Integer | Simulation starting save file |
```

Comment:
what does `Simulation starting save file` mean?

Reply:
Does "Save file from which to start simulation" make sense.

Reply:
so it's for restart purposes? if so, that should go in the description. and if so, is there a reason this is different from the other restart parameters/documentation we have?

Reply:
`t_step_start` isn't compatible with adaptive time-stepping because there's no explicit connection between # of time-steps and when the data is dumped.